text/utilities/0000-health-checker-buildpack.md

```diff
@@ -36,66 +36,62 @@ When conditions are met, the buildpack will do the following:
+1. For a Spring Boot app, run `pack build apps/maven -p target/demo-0.0.1-SNAPSHOT.jar -b paketo-buildpacks/java -b docker.io/paketo-buildpacks/health-checker -e BP_HEALTH_CHECKER_ENABLED=true -e THC_PORT='8080' -e THC_PATH='/actuator/health'` (it won't be necessary to include the `-b` arguments once the buildpack has been integrated with language family buildpacks). You'll see output like this at the end. It's installing the health checker and setting a process type.
 
     ```
     Paketo Health Checker Buildpack vDEVELOPMENT
       https://github.com/paketo-buildpacks/health-checker
       Build Configuration:
         $BP_HEALTH_CHECKER_DEPENDENCY  thc                    which health checker to contribute
         $BP_HEALTH_CHECKER_ENABLED     true                   contributes a health checker if enabled
-        $BP_HEALTH_CHECKER_THC_ARGS    8080 /actuator/health  arguments passed to tiny-health-checker, if used
-      Tiny Health Checker 0.4.0: Contributing to layer
-        Downloading from https://github.com/dmikusa-pivotal/tiny-health-checker/releases/download/v0.4.0/thc-x86_64-unknown-linux-musl
+      Tiny Health Checker 0.5.0: Contributing to layer
+        Downloading from https://github.com/dmikusa-pivotal/tiny-health-checker/releases/download/v0.5.0/thc-x86_64-unknown-linux-musl
         Verifying checksum
         Copying from /tmp/ce75bb97209981e03bf7e8aa52e2bfab78a50a44c7ed1787f4ace212711d61e5/thc-x86_64-unknown-linux-musl to /layers/paketo-buildpacks_health-checker/thc/bin
+        Writing env.launch/health-check/THC_PATH.default
+        Writing env.launch/health-check/THC_PORT.default
       Process types:
-    health-check: thc 8080 /actuator/health (direct)
+    health-check: thc (direct)
     ```
 
-2. You can run this with `docker run --health-cmd '/cnb/process/health-check' --health-interval 5s --health-timeout 2s -it apps/maven`. You could call the health check binary directly, but as you can have different health check dependencies that get installed by the buildpack, using the process type gives you more generic way to call the health check. It also allows for the image to bake in the health check arguments, rather than require the operator to set them. Plus, an operator could always call the binary directly if they wanted.
+2. You can run this with `docker run --health-cmd '/cnb/process/health-check' --health-interval 5s --health-timeout 2s -it apps/maven`. You could call the health check binary directly, but as you can have different health check dependencies that get installed by the buildpack, using the process type gives you more generic way to call the health check. It also allows for the image to bake in the health check configuration, rather than require the operator to set it. Plus, an operator can always override the configuration by setting the appropriate environment variables at runtime.
```

Comment:
```suggestion
2. You can run this with `docker run --health-cmd 'health' --health-interval 5s --health-timeout 2s -it apps/maven`. You could call the health check binary directly, but as you can have different health check dependencies that get installed by the buildpack, using the process type gives you more generic way to call the health check. It also allows for the image to bake in the health check configuration, rather than require the operator to set it. Plus, an operator can always override the configuration by setting the appropriate environment variables at runtime.
```

More extreme bikeshedding (feel free to ignore): We do put the processes on the front of the path and I wonder if we should be documenting the easier usage (w/o the absolute path) in more place

Reply:
✅  Ok, also good to know. Thanks